src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
```suggestion
  readonly shape: Shape = 'triangle';
```
I think the shape prop can be protected from changing.